admin/src/main/resources/io/buoyant/admin/js/router_clients.js

```diff
@@ -16,9 +16,15 @@ var RouterClients = (function() {
 
     routers.onAddedClients(addClients);
 
-    _.map(routers.clients(routerName), initializeClient);
+    var clients = routers.clients(routerName);
+    if (clients.length > 0) {
+      _.map(clients, initializeClient);
```

Comment:
This should probably be a forEach since we're not doing anything with the result? except that we should probably actually just assign routerClients to the result of _.map rather than having initializeClient push onto routerClients below.